### ISSUE TITLE
Login Feedback

### ISSUE THREAD

**Issue by senseiner** [LeafletBomb/leafletbomb#53]
Right now, I think there are several elements that do not make sense when you log in the application, these are my thoughts:

![login_feedback](https://user-images.githubusercontent.com/12564900/31570967-8f746734-b058-11e7-8378-8ae5ee6b9e1c.png)

**Comment by senseiner** [comment]
Instead of having everything expanded, you can have the log in centered and cleaner

Sign In

![sign_in](https://user-images.githubusercontent.com/12564900/31571012-7a7cecc4-b059-11e7-9853-f22eb4a8886d.png)

Sign Up

![sign_up](https://user-images.githubusercontent.com/12564900/31571017-873b5d06-b059-11e7-980f-017dc43be63e.png)

**Comment by bflanagin** [comment]
I agree, It was the scraps of what was left from the first mock-up. Its always been on my personal list, just wasn't as important as the overall functionality. I'll get what we have to something close to what you have here as soon as I am able.

**Comment by ryanleesipes** [comment]
From what I understand @senseiner can create some HTML/CSS. Perhaps you guys can do some pair programming on this one day.
